app/views/welcome/index.html.erb

```diff
@@ -1,13 +1,48 @@
 <!-- This area runs through the skills that have been entered in the database. To figure out correct divider for better looks -->
 <h3>Skills: </h3>
+<p>
 <% @skills.each_with_index do |t, index| %>
   <font style=" color: <%= t.rgb_for_current_rank(t.skill_rank - 1) %>;"><%= t.name %></font>
   <% if index == @skills.size - 1 %>
-    .
+    &nbsp;
     <% else %>
     -
   <% end %>
 <% end %>
+</p>
+
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-3">
+    </div>
+    <div class="col-md-6 offset-md-3">
+      <div align="center">
+        <table width="100%" class="table-bordered table-responsive">
+          <thead>
+              <th colspan="5" >Proficient
+                <div style="float: right;">Learning</div>
+              </th>
+          </thead>
+            <tr>
+              <!-- TODO This needs to be changed to a global variable-->
```

Comment:
I would be tempted to create a model looking something like this
what do you reckon?

```
class RankColours
  def all
    [ one, two, three ]
  end

  def rbg_for_rank(rank)
     all[rank]
  end
  
  def one
    "rgb(0, 153, 51)"
  end

  def two
    "rgb(13,201,247)" 
  end

  def three
    "rgb(127,127,127)" 
  end
end

```